homeassistant/components/nextcloud/sensor.py

```diff
@@ -8,7 +8,7 @@
 _LOGGER = logging.getLogger(__name__)
 
 
-def setup_platform(hass, config, add_entities, discovery_info=None) -> None:
+def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up the Nextcloud sensors."""
     if not discovery_info:
         return
```

Comment:
When debugging it appears that no discovery_info is passed into this setup_platform function.

Reply:
We're passing an empty dict as discovery_info. That's why we need to check for `None` as I wrote in my earlier comment.